src/test/integration/integrationtests.py

```diff
@@ -19,80 +19,127 @@
 
 
 class gateway_tests(unittest.TestCase):
-
     def test_services(self):
         buildings = get_buildings_with_services(services_url, access_token)
         query_params = {'page[size]': 500}
 
         for building_id in buildings:
             # Test that a building's service links back to the same building
-            building_object = id_request(locations_url, access_token, building_id)
-            for service in building_object['data']['relationships']['services']['data']:
-                service_object = id_request(services_url, access_token, service['id'])
-                parent_id = service_object['data']['relationships']['locations']['data'][0]['id']
+            building_object = id_request(locations_url, access_token,
+                                         building_id)
+            for service in building_object['data']['relationships'][
+                    'services']['data']:
+                service_object = id_request(services_url, access_token,
+                                            service['id'])
+                parent_id = service_object['data']['relationships'][
+                    'locations']['data'][0]['id']
```

Comment:
Thanks for taking care of the E502 (line too long) errors. There are still some (line 18, 39, 69, 73, 253, 271, 314, 327, 337, 574), could you also fix them? You could also use [backslashes](https://www.python.org/dev/peps/pep-0008/?#maximum-line-length) for the alignment.

Reply:
I was using [yapf](https://github.com/google/yapf) to to format the code to pep8 style, and it seemed to miss a lot of the line length errors. I'm not sure why. Even when I explicitly stated the column limit it still ignored it.
`python3 yapf -i --style='{based_on_style: pep8, column_limit: 79}'  ~/github/locations-frontend-api/src/test/integration/integrationtests.py`